include/oneapi/tbb/partitioner.h

```diff
@@ -358,7 +337,10 @@ struct proportional_mode : adaptive_mode<Partition> {
     template <typename Range>
     proportional_split get_split() {
         // Create a proportion for the number of threads expected to handle "this" subrange
```

Comment:
I think this comment needs to be updated, so that it explains the logic better, and - ideally - how it works for ranges with and without proportional split.